packages/mui-material/src/ButtonGroup/ButtonGroupButtonContext.ts

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+
+interface IButtonGroupButtonContext {
+  firstButton?: boolean;
+  lastButton?: boolean;
+  onlyChild?: boolean;
```

Comment:
`onlyChild` is redundant, isn't it? It could be derived from `firstButton && lastButton`

Reply:
You are correct. 👍